extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -137,6 +137,22 @@ public SecurityIdentity apply(Throwable throwable) {
         return performCodeFlow(identityProviderManager, context, resolver);
     }
 
+    private boolean isXHR(RoutingContext context) {
+        return "XMLHttpRequest".equals(context.request().getHeader("X-Requested-With"));
+    }
+
+    private boolean isAutoRedirectSet(TenantConfigContext configContext) {
```

Comment:
You can drop this function and check `configContext.oidcConfig.authentication.isXhrAutoRedirect()` directly, it is cleaner, true :-), but we access config directly all over the place

Reply:
Done

Reply:
@boosey, for some reasons I'm not seeing the change

Reply:
Ah. I misunderstood. I will change.